RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFitOOTPileupCorrection.h

```diff
@@ -42,16 +41,20 @@ class PulseShapeFitOOTPileupCorrection
 		     const std::vector<double> & its4Chi2, HcalTimeSlew::BiasSetting slewFlavor, int iFitTimes);
 
     const HcalPulseShapes::Shape* currentPulseShape_=nullptr;
+    const HcalTimeSlew* hcalTimeSlewDelay_=nullptr;
+    double TSdelay1GeV_=0;
 
-    void setPulseShapeTemplate  (const HcalPulseShapes::Shape& ps, bool isHPD, unsigned nSamples);
+    void setChi2Term( bool isHPD );
```

Comment:
this method is not used in this PR.
Why was it introduced?
I'm trying to avoid new non-const methods.